packages/wdio-shared-store-service/src/server.ts

```diff
@@ -46,6 +51,57 @@ export const startServer = () => new Promise<{ port: number, app: PolkaInstance
             store[key] = req.body.value as JsonCompatible | JsonPrimitive
             return res.end()
         })
+        .post('/setResourcePool', (req, res, next) => {
+            const key = req.body.key as string
+            const value = req.body.value as JsonCompatible | JsonPrimitive
+
+            if (!Array.isArray(value)) {
+                next(new Error('Resource pool must be an array of values'))
+                return
+            }
+
+            resourcePoolStore.set(key, value)
+            return res.end()
+        })
+        .post('/getValueFromPool/:key', async (req, res, next) => {
+            const key = req.params.key as string
+
+            if (!resourcePoolStore.has(key)) {
+                next(new Error(`'${key}' resource pool does not exist. Set it first using 'setResourcePool'`))
```

Comment:
@christian-bromann I had to change throwing to `next(new Error`, this is the correct way for polka to throw error messages

Reply:
sounds good, let's do:
```suggestion
                return next(new Error(`'${key}' resource pool does not exist. Set it first using 'setResourcePool'`))
```

Reply:
changed to 
```javascript
return next(`'${key}' resource pool does not exist. Set it first using 'setResourcePool'`)`
```
because new Error serializes to `{}`, serializing all errors by hand is too cumbersome
Polka recommends to just `next(string)` because this throws the error

I've added a catch + throw error on the client to read this message